app/views/papers/_links.html.erb

```diff
@@ -0,0 +1,28 @@
+<% paper = reference.paper %>
```

Comment:
Since `links` have their own controller now, we should put link related views in their own view folder.